cloud/smartos/imgadm

```diff
@@ -0,0 +1,146 @@
+#!/usr/local/bin/python
```

Comment:
I suppose that these module were written on FreeBSD, which uses /usr/local instead of /usr for 3rd party. There will be issues on other platforms, because not everyone is using python from /usr/local. I guess that best choice would be /usr/bin/env python